core/src/components/datetime/datetime.tsx

```diff
@@ -1414,7 +1422,13 @@ export class Datetime implements ComponentInterface {
    */
 
   private renderFooter() {
-    const { showDefaultButtons, showClearButton } = this;
+    const { disabled, readonly, showDefaultButtons, showClearButton } = this;
+    /**
+     * The cancel, clear, and confirm buttons
+     * should not be interactive if the datetime
+     * is disabled or readonly.
+     */
+    const isButtonDisabled = disabled || readonly;
```

Comment:
Now that the buttons are outright disabled, the datetime looks visually different with `readonly="true"`:

![disabled buttons](https://i.gyazo.com/6944294d9acf7e91a0645bfe916526b0.png)

This doesn't seem right? In all our other components, `readonly` doesn't alter the visuals. The prop's description also states that the datetime "appears normal but is not interactive." If disabling the buttons is the easiest way to achieve the functionality we want, one possibility is to manually reset the buttons' opacity.

Either way, the prop's description should probably also be updated, since it's now technically interactive (you just can't select new dates).

Reply:
I adjusted the description, but I had a tough time figuring out how to word it. I'm open to suggestions for further improvements.

Reply:
Maybe "the datetime appears normal but the selected date cannot be changed"?